addons/api/addon/models/credential-library.js

```diff
@@ -7,10 +7,19 @@ export const options = {
   http_method: ['GET', 'POST'],
 };
 
+// TODO: Remove `vault` type once we enable the feature
+export const TYPE_CREDENTIAL_LIBRARY_VAULT = 'vault';
```

Comment:
Do you have any opinions on this @randallmorey? Do you think it's worth keeping the old type? It's not easily "feature-flaggable" so it's not the best

Reply:
Will the three types coincide?  If not, this should be removed.  Generally this should reflect the state of the API.  However, since it's not yet merged on the backend, we should wait on merge just in case.